ontopy/ontology.py

```diff
@@ -353,23 +362,34 @@ def get_by_label(
 
         if colon_in_label is None:
             colon_in_label = self._colon_in_label
-        if colon_in_label:
-            if prefix:
-                raise ValueError(
-                    "`prefix` cannot be combined with `colon_in_label`"
+        if colon_in_label and prefix:
+            raise ValueError(
+                "`prefix` cannot be combined with `colon_in_label`"
+            )
+
+        splitlabel = label.split(":", 1)
+        if len(splitlabel) == 2 and not splitlabel[1].startswith("//"):
```

Comment:
`splitlabel[1].startswith("//")` will always be false, since `splitlabel[1]` has length one which never can starts with two characters...

What is the logics you want to express?